web/themes/new_weather_theme/templates/partials/daily-summary-list-item.html.twig

```diff
@@ -0,0 +1,131 @@
+            {% include "@new_weather_theme/partials/alert-link.html.twig" with { alertID: alertID, alertType: alertType, alertLevel: alertLevel} %}
+          </div>
+        {% endif %}
+      </div>
+    </div>
+    <div class="grid-row grid-gap">
+      {% for period in periods | filter(v => v) %}
+        {% set marginUnit = 3 %}
+        {% set tabletMarginUnit = 1 %}
+        {% set tabletCol = 6 %}
+        {%  if period.isOvernight %}
+          {% set label = "Overnight" | t %}
+          {% set lineColor = "midnight-indigo" %}
+          {% set tabletCol = 12 %}
+          {% set tabletMarginUnit = 3 %}
+        {% elseif period.isDaytime %}
+          {% set label = "Day" | t %}
+          {% set lineColor = "yellow" %}
+        {% else %}
+          {% set label = "Night" | t %}
+          {% set lineColor = "indigo" %}
+          {% set marginUnit = 1 %}
+        {% endif %}
+        <div class="wx-daily-forecast-summary-area grid-col-12 tablet:grid-col-{{ tabletCol }} margin-bottom-{{ marginUnit }} tablet:margin-bottom-{{ tabletMarginUnit }}">
+        <span class="display-block font-family-mono text-uppercase font-body-xs margin-bottom-1 text-primary-darker wx-marker-before wx-marker-before--{{ lineColor }}">
+            {{label}} {% if period.timeLabel %}<span class="daily-period-time-label text-gray-50">({{period.timeLabel}})</span>{% endif %}
+        </span>
+        <div class="display-flex flex-row flex-align-start">
+          {% if period.icon.base %}
+          <div class="width-5 height-5 margin-right-105">
+            <svg role="img" aria-hidden="true" class="width-full height-full">
+              <use xlink:href="{{ "/" ~ directory ~ "/assets/images/spritesheet.svg#" ~ period.icon.base }}"></use>
+            </svg>
+          </div>
+          {% endif %}
+          <div class="flex-1">
+            <p class="daily-short-forecast font-body-sm margin-top-2px margin-bottom-0 marker-height-sans-3">
+            {{period.shortForecast}}
+            </p>
+            {% if period.probabilityOfPrecipitation and period.probabilityOfPrecipitation > 1 %}
+            <p class="text-gray-50 font-body-xs margin-top-05 margin-bottom-0">
+            {{period.probabilityOfPrecipitation}}% {{ "chance of precipitation" | t }}
+            </p>
+            {% endif %}
+          </div>
+        </div>
+      </div>
+      {% endfor %}
+    </div>
+
+    {% set hourCount = dayHours | length %}
+    {% if hourCount > 0 %}
+        <div>
+            {#  Summary Temperature Chart #}
+            <div class="wx-chart-wrapper margin-top-3 margin-bottom-4">
+                <span class="font-family-mono text-primary-dark usa-sr-only">
+                    {{ "Temperature" | t }}
+                </span>
+                <div class="position-sticky left-0 display-flex flex-justify padding-top-2 padding-bottom-105 desktop:display-none">
```

Comment:
My mistake on this, we do need the arrows on desktop too because it doesn't the max amount of hours (only fits in the afternoon).